keras2onnx/ke2onnx/main.py

```diff
@@ -6,32 +6,32 @@
 from collections.abc import Iterable
 
 import numpy as np
-from ..proto import keras, is_tf_keras, is_keras_older_than
```

Comment:
I didn't change any of the imports, I just changed the order to follow PEP8.

Reply:
We don't enable PEP8 for this project, and the order here is organized by the category.

Reply:
OK, I'll revert it back.

Reply:
Done.